webapi/src/main/scala/org/knora/webapi/slice/admin/repo/service/KnoraProjectRepoLive.scala

```diff
@@ -0,0 +1,78 @@
+package org.knora.webapi.slice.admin.repo.service
+
+import play.twirl.api.TxtFormat
+import zio._
+
+import dsp.valueobjects.V2
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.HasSelfJoinEnabled
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectDescription
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectKeyword
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectLogo
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectLongname
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectShortcode
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.ProjectShortname
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.Status
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.messages.store.triplestoremessages.BooleanLiteralV2
+import org.knora.webapi.messages.store.triplestoremessages.SparqlExtendedConstructResponse.ConstructPredicateObjects
+import org.knora.webapi.messages.store.triplestoremessages.StringLiteralV2
+import org.knora.webapi.messages.store.triplestoremessages.SubjectV2
+import org.knora.webapi.messages.twirl.queries.sparql.admin.txt.getProjects
+import org.knora.webapi.slice.admin.domain.model.KnoraProject
+import org.knora.webapi.slice.admin.domain.service.KnoraProjectRepo
+import org.knora.webapi.slice.common.service.PredicateObjectMapper
+import org.knora.webapi.slice.ontology.domain.service.OntologyRepo
+import org.knora.webapi.slice.resourceinfo.domain.InternalIri
+import org.knora.webapi.store.triplestore.api.TriplestoreService
+
+final case class KnoraProjectRepoLive(
+  private val triplestore: TriplestoreService,
+  private val pom: PredicateObjectMapper
+) extends KnoraProjectRepo {
+
+  override def findById(id: InternalIri): Task[Option[KnoraProject]] =
+    findOneByQuery(getProjects(maybeIri = Some(id.value), None, None))
+
+  override def findByProjectIdentifier(id: ProjectIdentifierADM): Task[Option[KnoraProject]] = {
```

Comment:
The naming here is a bit inconsistent: `findById` vs. `findByProjectIdentifier`. I would use either both names with or both names without "project".

Reply:
=> 1a28bc980ad2f80798a134c1195a9f40cc7cadc0